repository.yaml

```diff
@@ -1,4 +1,9 @@
+digital-service: Central Reference Data Library Cache
+description: This service provides an API and caching layer for reference data hosted at DPS (Data Platform Services).
 repoVisibility: public_0C3F0CE3E6E6448FAD341E7BFA50FCD333E06A20CFF05FCACE61154DDBBADF71
 type: service
 service-type: backend
-tags: []
\ No newline at end of file
+tags: ['api']
```

Comment:
Hmm I'm not sure if this should be tagged `api` - I think this might imply that it's a public API and tell the catalogue to fetch information that's only available in public API repositories